lib/pwnlib/tubes/serialtube.rb

```diff
@@ -0,0 +1,111 @@
+# encoding: ASCII-8BIT
+
+require 'rubyserial'
+
+require 'pwnlib/tubes/tube'
+
+module Pwnlib
+  module Tubes
+    # Serial Connections
```

Comment:
change here to
```ruby
# @!macro [new] raise_eof
#   @raise [Pwnlib::Errors::EndOfTubeError]
#     If the request is not satisfied when all data is received.

# Serial Connections
class SerialTube < Tube
```
otherwise YARD will have wrong result